File: Sample/java/src/main/java/net/nend/sample/java/banner/layoutpatterns/SecondFragment.java
package net.nend.sample.java.banner.layoutpatterns;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

import net.nend.sample.java.R;

public class SecondFragment extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, 
            ViewGroup container, Bundle savedInstanceState) {

        return inflater.inflate(R.layout.replace_fragment2, null);
    }
}

File: Sample/kotlin/src/main/java/net/nend/sample/kotlin/banner/SecondFragment.kt
package net.nend.sample.kotlin.banner

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import net.nend.sample.kotlin.R

class SecondFragment : androidx.fragment.app.Fragment() {

    override fun onCreateView(inflater: LayoutInflater,
                              container: ViewGroup?, savedInstanceState: Bundle?) =
            View.inflate(context, R.layout.replace_fragment2, null)
}